tests/test_client.py

```diff
@@ -0,0 +1,158 @@
+from unittest import mock
+
+import pytest
+import rdflib
+
+from conftest import skip_if_nanopub_server_unavailable
+from nanopub import NanopubClient, namespaces, Nanopub
+
+client = NanopubClient(use_test_server=True)
+
+TEST_ASSERTION = (namespaces.AUTHOR.DrBob, namespaces.HYCL.claims, rdflib.Literal('This is a test'))
+
+TEST_ORCID_ID = 'https://orcid.org/0000-0000-0000-0000'
+
+
+def _get_mock_profile():
+    mock_profile = mock.MagicMock()
+    mock_profile.get_orcid_id.return_value = TEST_ORCID_ID
+
+    return mock_profile
+
+
+@mock.patch('nanopub.nanopub.profile', _get_mock_profile())
```

Comment:
This is exactly the same as in test_nanopub.py, but I will create a separate PR to fix this in a nice way (by just setting up the profile in the github action)

Reply:
OK!